apps/worker/main.py

```
@@ -0,0 +1,171 @@
+        """
+        try:
+            # Poll queue with timeout
+            result = await self.redis_client.blpop(
+                "queue:extraction", timeout=self.poll_timeout
+            )
+
+            if not result:
+                # No job available
+                return
+
+            queue_name, job_data = result
```

Ruff (0.14.1)

Unpacked variable `queue_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)